recipe/meta.yaml

```
@@ -189,6 +193,52 @@ outputs:
         - lit -h
         - llvm-lit -h
 
+  - name: llvmlite
+    build:
+      number: {{ build + 100 }}
+      string: "llvmstatic_h{{ PKG_HASH }}_{{ build + 100 }}"
+      skip: true  # [py<310]
+      script_env:
+        - PY_VCRUNTIME_REDIST
+      script: {{ RECIPE_DIR }}/llvmlite/build.sh  # [unix]
+      script: {{ RECIPE_DIR }}/llvmlite/bld.bat   # [win]
+      ignore_run_exports:
+        - libllvm{{ major_ver }}  # [unix and not ppc64le]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make                                   # [unix]
+      host:
+        - python
+        - setuptools
+        - {{ pin_subpackage("llvm-tools", exact=True) }}
+        - {{ pin_subpackage('llvmdev', exact=True) }}
+        - {{ pin_subpackage('llvm', exact=True) }}
```

These `exact=True` get in the way. You need to express them like:

```suggestion
        - llvm-tools {{ PKG_VERSION }}  *_{{ build }}
        - llvmdev {{ PKG_VERSION }}  *_{{ build }}
        - llvm {{ PKG_VERSION }}  *_{{ build }}
```

I think?

---

I did this and I got the sha mismatch error for all builds. I'll try again though.